doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+> **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+-	Control, User and Management traffic types
+-	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+
+•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+
+•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•	**Workload diversity**: Should be capable to handle any VNF , CNF and BareMetal workloads.
+
+•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
+
+•	**Future proof**: Should support known and emerging acceleration technologies e.g. SmartNICs, FPGAs and Programmable Switches. 
+
 
 <a name="5.1.3"></a>
-### 5.1.3 Networking Requirements
+### 5.1.3 Network Layering and Concepts
+
+Cloud and Telco networking are layered, and it is very important to keep the layering dependencies low to enable security, separation and portability in between multiple implementations.
+
+Before we start developing a deep model we need to agree on some foundational concepts and layering that allow decoupling of implementations in between the layers. We will emphasize four concepts in this section
+
+ - Underlay and Overlay Networking concepts
+ - Hardware and Software Infrastructure Layer concepts
+ - Software Defined Networking, SDNu and SDNo concepts
+ - Programmable Networking Fabric concept
+
+<a name="5.1.3.1"></a>
+#### 5.1.3.1 Underlay and Overlay Networking concepts
+
+The ETSI NFV model divide networking in an Underlay and an Overlay Network layer. The purpose with this layering is to ensure separation of the SW Virtualisation tenants (Workload) Overlay Networks from each other, whilst allowing the traffic to flow on the shared Underlay Network in between all Ethernet connected HW units.
+
+The Overlay Networking separation is often done through encapsulation of Tenants traffic using overlay protocols e.g. through VxLAN or EVPN on the Underlay Networks e.g. based on L2 (VLAN) or L3 (IP) networks.
```

Comment:
"...encapsulation of Tenants traffic..." - Tenants is possessive. Pls amend "Tenants" -> " Tenants' "